ci-operator/step-registry/origin/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- smarterclayton
+- wking
+- stevekuznetsov
```

Comment:
Also I don't really want to be in these -- I don't own the content. We're at the top-level anyway for approval